cmd/cdi/cmd/validate.go

```diff
@@ -40,6 +41,13 @@ were reported by the registry.`,
 			return
 		}
 
+		fmt.Printf("CDI Registry has errors:\n")
+		for path, specErrors := range cdiErrors {
+			fmt.Printf("Spec file %s:\n", path)
+			for idx, err := range specErrors {
+				fmt.Printf("  %d: %v\n", idx, strings.TrimRight(err.Error(), "\n"))
```

Comment:
What about just:
```suggestion
				fmt.Printf("  %d: %v\n", idx, strings.Trim(err.Error()))
```
We should be ok to just ignore all leading and trailing whitespace in error messages.

Reply:
Makes sense. Changed it to `strings.TrimSpace()`